src/Login/Login.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import './Login.css'
+
+class Login extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      login: '',
+      password: ''
+    }
+  }
+
+  authFormSubmitHandler = (event) => {
+    event.preventDefault()
+    localStorage.setItem('token', process.env.REACT_APP_TOKEN)
+    const { history } = this.props
+    history.push('/departments')
```

Comment:
you should not use history for such purposes. 
Use <Redirect> component instead

Reply:
do you mean redirect do Departments component?

Reply:
Yes